src/variables/utils/astim.ts

```diff
@@ -19,11 +19,19 @@ const parseAllVariables = (ast: File): MemberExpression[] => {
 }
 
 export const parseASTIM = (query: string): ASTIM => {
-  const ast: File = parse(query)
-  // Using `any` to circumvent ts error
-  const variables: any = ast ? parseAllVariables(ast) : []
+  let ast: File = null
+  try {
```

Comment:
This method `parseASTIM()` is called in the existing DataExplorer, as well as when opening a cell on the Dashboard.